tests/test_s3_to_json_s3.py

```diff
@@ -599,9 +605,10 @@ def test_is_expected_validation_error(self):
                     "'stepPath' is a required property",
                 ]
         }
-        assert s3_to_json_s3.is_expected_validation_error(
+        unexpected_validation_errors = s3_to_json_s3.remove_expected_validation_errors(
                 validation_result=validation_result,
                 client_info=client_info)
+        assert unexpected_validation_errors == {}
```

Comment:
Should there be a test for an example with duplicated validation errors or is there code that drops the duplicates mentioned here: [ETL-358](https://sagebionetworks.jira.com/browse/ETL-358)?

Reply:
Good idea. Added.